src/components/SingInForm.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { Link } from "react-router-dom";
+import { toast } from "react-toastify";
+import { Button, Form, FormInput } from "semantic-ui-react";
+import { singInRequest } from "../api";
+import { setAutorizationStatusAction } from "../redux/actions/autorizationAction";
+
+export default function SingInForm(props: { history: string[] }): JSX.Element {
+  const dispatch = useDispatch();
+  const [userLogin, setUserLogin] = useState("");
+  const [userPassword, setUserPassword] = useState("");
+  let userDataFromDataBase;
+
+  async function checkUserData() {
+    const promise = await singInRequest(userLogin);
```

Comment:
wrong variable name.

Reply:
I change to "checkUserPassword"

Reply:
const promise - is wrong. Because it;s not promise actually. Since you awaited it

Reply:
will be "userData"